src/day8/re_frame_10x/panels/traces/subs.cljs

```diff
@@ -116,13 +116,16 @@
   (fn [[traces categories] _]
     (filter (fn [trace] (when (contains? categories (:op-type trace)) trace)) traces)))
 
-(defn query->fn [query]
-  (if (= :contains (:type query))
-    (fn [trace]
-      (string/includes? (string/lower-case (str (:operation trace) " " (:op-type trace)))
-                        (:query query)))
-    (fn [trace]
-      (< (:query query) (:duration trace)))))
+(defn query->fn [trace query]
+  (cond
+    (= :contains (:type query))
+    (string/includes? (string/lower-case (str (:operation trace) " " (:op-type trace)))
```

Comment:
it seems that this form could be refactored out `(string/includes?...` into a binding or a fn as it is repeated twice and is quite big